platform/osx/os_osx.mm

```diff
@@ -602,8 +602,8 @@ - (void)mouseMoved:(NSEvent *)event {
 	mm->set_position(pos);
 	mm->set_global_position(pos);
 	Vector2 relativeMotion = Vector2();
-	relativeMotion.x = [event deltaX] * OS_OSX::singleton->_mouse_scale([[event window] backingScaleFactor]);
-	relativeMotion.y = [event deltaY] * OS_OSX::singleton->_mouse_scale([[event window] backingScaleFactor]);
+	relativeMotion.x = [event deltaX] * OS_OSX::singleton -> _mouse_scale([[event window] backingScaleFactor]);
+	relativeMotion.y = [event deltaY] * OS_OSX::singleton -> _mouse_scale([[event window] backingScaleFactor]);
```

Comment:
damn it.
pre commit hook only let me commit like this ...
Is this because of clang 7.0 ? (what I have locally installed (I think))

Reply:
I had this problem too when I saved with auto format on with my IDE (but only for osx). But I don't use any pre commit hook, then I just fixed and push again.

Reply:
I just went through the painful process of installing clang 5.0.2 on OS X
I wanted to do it with brew to be able to switch clang-format ....(version)

but there was no formula.

no I realised that I could just download the whole llvm from the official site, than copy the clang format bin and put it in Cellar/clang-format/5.0.2/bin and brew can handle it perfectly fine!!!

Reply:
nice this feels sooo good!!! (to have a working clang installation. the pre commit hook now is not even called anymore since it formatted correctly on save)